### DISCUSSION TITLE
How can i see the onboarding screens again?

### DISCUSSION THREAD

**Discussion by ghost** [GradienceTeam/Gradience#645] [Q&A] [answered]
I didn't find the Gradience config files inside my User Folder.
How can i see the onboarding screens again? I passed them too quickly. I'm on Manjaro. Thank you.

**Comment by 0xMRTT** [comment, marked as answer]
@daudix-ufo maybe we should add the ability to do the onboarding again

@Greelings you can go in dconf editor to the path `/com/github/GradienceTeam/Gradience` and set the key `first-run` to `true`

**Reply by ghost** [reply]
It worked. Thx.

**Reply by daudix** [reply]
Hmm, I thought about "Factory Reset" button in settings which will also display welcome screen on next launch, but this is nice idea!

Also, I can add instructions in README on how to re-run setup 😉

**Reply by daudix** [reply]
Here we go!

https://github.com/GradienceTeam/Gradience#%EF%B8%8F-miscellaneous
